chef-dk.gemspec

```diff
@@ -53,7 +55,7 @@ Gem::Specification.new do |gem|
 
   gem.add_dependency "chef-provisioning", "~> 1.2"
 
-  gem.add_development_dependency "github_changelog_generator", "1.11.3"
+  gem.add_development_dependency "github_changelog_generator"
```

Comment:
There's no such thing as spontaneous combustion when you check in Gemfile.locks via PR :) The most constraint anyone should do is `>=` in this world, until we are sure a higher version will fail.